packages/react/src/components/Nav/Nav.base.tsx

```diff
@@ -386,6 +386,9 @@ export class NavBase extends React.Component<INavProps, INavState> implements IN
   }
 
   private _isGroupExpanded(group: INavLinkGroup): boolean {
+    if (group.isCollapsed !== undefined) {
+      return group.isCollapsed;
+    }
```

Comment:
Since this is currently named `isCollapsed`, the `isGroupExpanded` function should return `!group.isCollapsed`, otherwise it's backwards 😄. Though if you rename it, this logic is good as-is.